TestFoundation/TestNSOperationQueue.swift

```diff
@@ -36,17 +41,21 @@ class TestNSOperationQueue : XCTestCase {
 
     func test_OperationPriorities() {
         var msgOperations = [String]()
-        let operation1 : BlockOperation = BlockOperation(block: {
+        let operation1 = BlockOperation(block: {
             msgOperations.append("Operation1 executed")
+            sleep(1)
```

Comment:
Added time-lag to resolve the test-case failures seen sometimes on TestFoundation.